testing/smoke/supported-os/main.tf

```diff
@@ -164,6 +175,19 @@ resource "aws_instance" "apm" {
   vpc_security_group_ids = [aws_security_group.main.id]
 }
 
+resource "null_resource" "getapmserverlogs" {
```

Comment:
I haven't tested this, will it always run before the `aws_instance` is destroyed?

Reply:
```suggestion
resource "null_resource" "apm_server_log" {
```